README.md

```diff
@@ -1,47 +1,16 @@
-# BepInEx LoveMachine ([日本語](マニュアル.md))
-[![.NET](https://github.com/Sauceke/BepInEx.LoveMachine/actions/workflows/commit.yml/badge.svg)](https://github.com/Sauceke/BepInEx.LoveMachine/actions/workflows/commit.yml)
-[![Download](https://img.shields.io/github/downloads/Sauceke/BepInEx.LoveMachine/total)](https://github.com/Sauceke/BepInEx.LoveMachine/releases/latest/download/LoveMachineInstaller.exe)
+# BepInEx LoveMachine ([日本語](Documentation/Japanese))
```

Comment:
This link will not work when deployed to Github Pages.

Reply:
Right, I'll wait for your response to the separate github pages / readme approach.